### PR TITLE
(node/lsstcam-fcs.cp.lsst.org) deploy host as " EL7 "

### PR DESCRIPTION
**Pull request by dtapiacl**

### CODE REVIEW

File: hieradata/node/lsstcam-fcs.cp.lsst.org.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+network::interfaces_hash:
+  eno1:  # fqdn
+    bootproto: "none"
+    ipaddress: "139.229.175.82"
```
Comment (jhoblitt) [inline hieradata/node/lsstcam-fcs.cp.lsst.org.yaml:5]: The IP is in ipam but not the device.

**Review by jhoblitt — CHANGES_REQUESTED** [review]

**Comment by glennmorris** [comment]
Added a couple of commits to enable the ccs canbus module.
The Puppetfile bump will lead to a trivial merge conflict at some point, sorry.
It requires a private hiera change too.

File: hieradata/node/lsstcam-fcs.cp.lsst.org.yaml (hunk 1 above)
Comment (jhoblitt) [reply hieradata/node/lsstcam-fcs.cp.lsst.org.yaml:5]: This address doesn't appear to be associated with a device.
image

**Review by jhoblitt — CHANGES_REQUESTED** [review]

**Review by jhoblitt — CHANGES_REQUESTED** [review]
Please squash 4db641ec81840968b29d08290368ebead1fca360

File: hieradata/node/lsstcam-fcs.cp.lsst.org.yaml (hunk 1 above)
Comment (dtapiacl) [reply hieradata/node/lsstcam-fcs.cp.lsst.org.yaml:5]: Device is updated on ipam.cp.lsst.org . I think this is an issue between IPAMs as i have been updating CP only and LS contains outdated information.

**Comment by dtapiacl** [comment]
> Please squash [4db641e](https://github.com/lsst-it/lsst-control/commit/4db641ec81840968b29d08290368ebead1fca360)

Commit was squashed on to: [2274c33](https://github.com/lsst-it/lsst-control/pull/1158/commits/2274c33a0d515dfb7f88d932ac35d09d4d000cce)

**Review by jhoblitt — APPROVED** [review]

**Pull request merged**